core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -1135,8 +1134,7 @@ final class ShuffleBlockFetcherIterator(
       } else {
         sendRequest(request)
       }
-      numBlocksInFlightPerAddress(remoteAddress) =
-        numBlocksInFlightPerAddress.getOrElse(remoteAddress, 0) + request.blocks.size
+      numBlocksInFlightPerAddress.getOrElse(remoteAddress, 0) += request.blocks.size
```

Comment:
It doesn't seem like this can work/compile? the result of getOrElse is an int. Not sure if we can optimize this one.
The rest looks fine.

Reply:
It is my bad, has updated. Thanks @srowen